src/sagemaker/jumpstart/accessors.py

```diff
@@ -10,6 +10,7 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+# pylint: skip-file
```

Comment:
Lets make sure we follow up on fixing these lints instead of doing `pylint: skip-file`